tdms/tests/matlab/run_tests.m

```diff
@@ -0,0 +1,15 @@
+import matlab.unittest.TestRunner;
+import matlab.unittest.Verbosity;
+import matlab.unittest.constraints.ContainsSubstring;
+import matlab.unittest.selectors.HasBaseFolder;
+
+addpath(genpath('.'));
+
+suite = testsuite(pwd, 'IncludeSubfolders', true);
+suite = suite.selectIf(HasBaseFolder(ContainsSubstring('tests')));
+
+runner = TestRunner.withTextOutput('OutputDetail', Verbosity.Detailed);
+results = runner.run(suite);
+
```

Comment:
If you wnat to add code coverage (here or another PR):

https://github.com/dstansby/matlab-sandbox/blob/6da492672f240df162810cf9948b4a2bc8377841/run_tests.m#L22

Reply:
At this state I'm not too worries about adding codecov, but would definitely be nice in the future